docs/release-notes/NuGet-6.1.md

```diff
@@ -101,6 +101,16 @@ NuGet distribution vehicles:
 
 **[List of all issues fixed in this release - 6.1](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=Z2lkOi8vcmFwdG9yL1JlbGVhc2UvNjY5ODY)**
 
+## Known issues
+
+### dotnet nuget push -n|--no-symbols or -d|--disable-buffering raises `error: File does not exist ...` exception. - [#10601](https://github.com/NuGet/Home/issues/11601)
```

Comment:
Update correct number and we're good to merge.

Reply:
Done.